indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+                  <td id='earthT' align='center' width='110' >&nbsp;</td>
+                  <td id='plutoT' align='center' width='110' >&nbsp;</td>
+                  <td id='mercuryT' align='center' width='110' >&nbsp;</td>
+                  <td id='venusT' align='center' width='110' >&nbsp;</td>
+                  <td id='moonT' align='center' width='110' >&nbsp;</td>
+                  </tr>
+                  </table>
+            </div> <!-- end nav -->
+            <div id="col1">
+                  <div id="col1content">
+                        <h3 id='orderform'>Flights to Mars - Order Form</h3>
+                        <p id='details'>
+                        <form method=post action="jcform.asp" name="edgar" >
+                              <p id='pt1'> First name
+                              <input type=text id="name1" ></p>
+                              <p id='pt2'> Address
+                              <input type=text id="address"  ></p>
+                              <p id='pt3'> Phone Number
+                              <input type=text id="phone" ></p>
+                              <p id='pt4'> Email
+                              <input type=text id="email" ></p>
+                              <p>Credit card type
+                              <input type=Radio name="CC" value="Visa" checked>Visa
+                              <input type=Radio name="CC" value="Amex">Amex
+                              <input type=Radio name="CC" value="Mastercard">Mastercard
+                              </p>
+                              <p>Credit card number
+                              <input type=Password id="cardnumber" >
+                              </p>
+                              <p><nobr><input type='submit' value="Buy">
+                              <input type='reset' value="Clear"></nobr></p>
+                        </form></p>
+                  </div>  <!-- end col1content -->
+            </div>  <!-- end col1 -->
+            <div id="col2">
+                  <div id="col2content">
+                  <h3 id='intro'>Introduction </h3>
+                  <p id='blurb'>
+                  Do you want to be amongst the first space flight passengers
+                  to Mars? Join our flight service (as endorsed by Patrick
+                  Mooreish) and see the Red Planet for yourselves !!
+                  All you have to do is fill in the details (Name, Address,
+                  Phone, Email and Credit Card details) and we will rush you
+                  the tickets. This flight is SO NOT likely to happen !!!!!! (for a while)</p>
+                  </div>  <!-- end col2content -->
+            </div>  <!-- end col2 -->
+            <div id="footer">
+            <p><b>&copy; Bear Computer Services Ltd. | Original style design by BlueSky Web Studios</b></p>
+            </div>  <!-- end footer -->
+      </div>  <!-- end wrapper -->
+</body>
+<script>

```

Comment:
Good work here I would have been more intrigued to apply a modular approach and keep javascripts in separate folder and load them when needed.